main/src/com/google/refine/expr/functions/strings/Phonetic.java

```diff
@@ -72,7 +72,10 @@ public Object call(Properties bindings, Object[] args) {
                     return new EvalError(ControlFunctionRegistry.getFunctionName(this)
                             + " expects a string for the second argument");
                 }
-            }
+            } else {
+                return new EvalError(ControlFunctionRegistry.getFunctionName(this)
+                        + " expects a string for the second argument.Make sure to pass the argument in proper string format");
+                }     
```

Comment:
The indentation seems a bit off: this "}" should be at the same level as the first "}" of "} else {".